### PR TITLE
fix(pointcloud) : layer.visible and PointsMaterial.copy

### PR DESCRIPTION
**Pull request by mbredif**
## Description
post review fixes of #764 

## Motivation and Context
see #764

### CODE REVIEW

**Review by peppsac — APPROVED** [review]
Thanks for the quick fixes.
One side note: the git commit messages should contain the rationale of the changes, not just a link to another issue/PR (to avoid depending to much on the code hosting solution).

**Comment by peppsac** [comment]
(and fix the commit title which doesn't respect #518 as well)

**Comment by mbredif** [comment]
okay, okay... :)

**Comment by peppsac** [comment]
> okay, okay... :)

Believe it or not but commit messages are useful :)

File: src/Renderer/PointsMaterial.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ class PointsMaterial extends RawShaderMaterial {
         this.uniforms.overlayColor.value = this.overlayColor;
     }
 
-    copy(source) {
+    update(source) {
```
Comment (autra) [inline src/Renderer/PointsMaterial.js:47]: I remember you mentioned calling the super method didn't work... Did you have the chance to investigate why @mbredif?

**Pull request merged**